bundles/core/org.eclipse.smarthome.core.audio/src/main/java/org/eclipse/smarthome/core/audio/internal/AudioManagerImpl.java

```diff
@@ -81,18 +96,45 @@ public void play(AudioStream audioStream) {
     }
 
     @Override
-    public void play(AudioStream audioStream, String sinkId) {
+    public void play(final AudioStream audioStream, String sinkId) {
         if (audioStream != null) {
             AudioSink sink = getSink(sinkId);
+            Set<String> sinks = this.getGroup(sinkId);
 
             if (sink != null) {
                 try {
                     sink.process(audioStream);
+                    return;
                 } catch (UnsupportedAudioFormatException e) {
                     logger.error("Error playing '{}': {}", audioStream.toString(), e.getMessage());
                 }
+            } else if (sinks.size() > 0) {
+                for (final String aSink : sinks) {
+                    // SafeMethodCaller.call(new SafeMethodCaller.Action<Void>() {
+                    // @Override
+                    // public Void call() {
+                    // logger.debug("Playing {} on {}", audioStream.toString(), aSink);
+                    // long stamp = System.currentTimeMillis();
+                    // play(audioStream, aSink);
+                    // logger.debug("Call completed in {} ms", System.currentTimeMillis() - stamp);
+                    // return null;
+                    // }
+                    // }, 10000);
+                    audioPool.submit(new Runnable() {
+
+                        @Override
+                        public void run() {
+                            logger.debug("Playing {} on {}", audioStream.toString(), aSink);
+                            long stamp = System.currentTimeMillis();
+                            play(audioStream, aSink);
+                            logger.debug("Playing {} on {} completed in {} ms", audioStream.toString(), aSink,
+                                    System.currentTimeMillis() - stamp);
+                        }
+                    });
+                }
             } else {
-                logger.warn("Failed playing audio stream '{}' as no audio sink was found.", audioStream.toString());
+                logger.warn("Failed playing audio stream '{}' as no audio sink or group was found.",
+                        audioStream.toString());
```

Comment:
That has not been introduced by this PR but as you changed that line already you could fix the implicit `toString()` function call.